packages/app/src/components/SideBarNavigation/index.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Sidebar } from 'react-rainbow-components';
+import { useLocation } from 'react-router-dom';
+import StyledContainer from './styled';
+
+const SideBarNavigation = (props) => {
+    const { className, style, children } = props;
+    const location = useLocation();
+    const routes = [];
+    const arrayChildren = React.Children.toArray(children);
+    arrayChildren.forEach((element, index) => {
```

Comment:
let's use a `map` here instead of a `forEach`
```js
const routes= arrayChildren.map(() => {});
```

Reply:
also, I don't think we need to implement that logic our self here, take a look at https://reacttraining.com/react-router/web/api/Hooks/useroutematch ... `match` API will solve the problem of knowing which item is selected based on the current location

And navigation could be made by using `useHistory` hook then history.push(path)

Reply:
> also, I don't think we need to implement that logic our self here, take a look at https://reacttraining.com/react-router/web/api/Hooks/useroutematch ... `match` API will solve the problem of knowing which item is selected based on the current location

nice tip, thanks